docs/editors/vim.md

```diff
@@ -303,7 +303,7 @@ open buffer.
 
 ## Worksheets
 
-Metals allows users to create a `*.worksheet.sc` file and see evaluations right
+Metals allows users to create a `src/main/scala/*.worksheet.sc` file and see evaluations right
```

Comment:
So I actually wouldn't change this, or at least not without further explanation. Worksheets actually still work at the root of your project. They won't have access to everything in your package like they would if they were in `src/main/scala`, but the worksheet still works and evaluates. So if you are going to change this at all, you may want to make that distinction.